project.clj

```diff
@@ -1,11 +1,13 @@
-(defproject com.oscaro/tools-io "0.3.21"
+(defproject com.oscaro/tools-io "0.3.22-SNAPSHOT"
   :description "Oscaro's generic I/O tools collection"
   :url "https://github.com/oscaro/tools-io"
   :license {:name "Eclipse Public License"
             :url  "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.10.3"]
                  [cheshire "5.10.0"]
-                 [clj-commons/clj-yaml "0.7.106"]
+                 [circleci/clj-yaml "0.6.0"]
+                 [clj-commons/clj-yaml "0.7.106"]              
+                 [org.apache.commons/commons-compress "1.20"]
```

Comment:
this library is relatively fat, maybe we want to keep tools-io lightweight ?

Reply:
agree with that. The best way to do it is to make a `register-file-encoding!` in the same way than the existing `register-file-pred!` and make a separate project `tools-io-bz2`